TST: test_arima111_predict_exog_2127  fails again

after adding a conda update in #4261, the test fails again on one machine.
We didn't have any problems with it since my last fix to it in #4004

In #4004 I added an assert for "converged" which does not fail here. maxiter is large so there "should not" be a problem with the estimation. However, something is still strange and fragile in this test.



```
E                   AssertionError: 
E                   Not equal to tolerance rtol=1e-07, atol=5e-06
E                   
E                   (mismatch 100.0%)
E                    x: array([ 0.025868,  0.023159,  0.02433 ,  0.023635,  0.023856,  0.02367 ,
E                           0.023702,  0.023624,  0.023595,  0.023578,  0.023572])
E                    y: array([ 0.025911,  0.023213,  0.024366,  0.023688,  0.023898,  0.02372 ,
E                           0.023748,  0.023674,  0.023644,  0.023629,  0.023623])
```

--- a/statsmodels/tsa/tests/test_arima.py
+++ b/statsmodels/tsa/tests/test_arima.py
@@ -3,6 +3,7 @@ from statsmodels.compat.testing import skip, skipif
 
 import os
 import warnings
+from distutils.version import LooseVersion
 
 import numpy as np
 from numpy.testing import (assert_almost_equal, assert_, assert_allclose,
@@ -20,6 +21,9 @@ from statsmodels.tsa.arima_model import ARMA, ARIMA
 from statsmodels.regression.linear_model import OLS
 from statsmodels.tsa.tests.results import results_arma, results_arima
 from statsmodels.tsa.arima_process import arma_generate_sample
+
+import scipy  # only needed for version check
+scipy_old = LooseVersion(scipy.__version__) < '0.16'
 
 try:
     import matplotlib.pyplot as plt
@@ -2054,7 +2058,7 @@ def test_arima_diff2():
                      229.457]
     assert_almost_equal(predicted, predicted_res, 3)
 
-
+@skipif(scipy_old, reason='scipy is old, test might fail')
 def test_arima111_predict_exog_2127():
     # regression test for issue #2127
     ef =  [ 0.03005,  0.03917,  0.02828,  0.03644,  0.03379,  0.02744,

